pkgs/applications/audio/bitwig-studio/bitwig-studio1.nix

```diff
@@ -77,7 +73,7 @@ stdenv.mkDerivation rec {
       -not -path '*/resources/*' | \
     while IFS= read -r f ; do
       wrapProgram $f \
-        --prefix PATH : "${binPath}" \
+        --suffix PATH : "${lib.makeBinPath [ xdg-utils zenity ffmpeg ]}" \
```

Comment:
Since this PR is focused on `xdg-utils`, I think we should still leave the non `xdg-utils` options with `--prefix`. I think suffix should only really be used for "system dependencies" / anything that doesn't have a predictable runtime behaviour.